src/ChildProcessUtilities.js

```diff
@@ -1,4 +1,5 @@
 import child from "child_process";
+import objectAssign from "object-assign";
```

Comment:
I guess it depends on the runtime - `object-assign` was available already, so I just used it.